cli/uninstall/uninstall.go

```diff
@@ -111,6 +111,11 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 		if programVersion, err = getDefault(program, binDst); err != nil {
 			return err
 		}
+	} else if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] != 'v' {
```

Comment:
https://github.com/tarantool/tt/blob/e90220698092b936858d8ed34b101b0b911fe6a5/cli/install/install.go#L498

For fully symmetry, we need to try to delete two versions (with 'v' and without). Since the `install` could install `1.2.3-xxx-foo-bar` if the tag exist.

Reply:
Fixed.